setuptools_odoo/make_default_setup.py

```diff
@@ -120,6 +121,55 @@ def make_default_setup_addons_dir(addons_dir, force,
                                  odoo_version_override)
 
 
+def clean_setup_addons_dir(addons_dir):
+    paths_to_remove = []
+
+    for addon_name in os.listdir(addons_dir):
```

Comment:
rather enumerate `setup/*/{odoo_addons,odoo}`?

Reply:
Indeed, that would be better in case the addon directory has been deleted but not the setup folder. I will change it.